### PR TITLE
Onboarding static sites

### PR DESCRIPTION
**Pull request by khaled-zayed**

### CODE REVIEW

File: data/templates/Detector_StaticSites.csx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+
+[ArmResourceFilter(provider: "Microsoft.Web", resourceTypeName: "staticSites")]
+[Definition(Id = "YOUR_DETECTOR_ID", Name = "", Author = "YOUR_ALIAS", Description = "")]
+public async static Task<Response> Run(DataProviders dp, OperationContext<ArmResource> cxt, Response res)
+{
+    res.Dataset.Add(new DiagnosticData()
+    {
+        Table = await dp.Kusto.ExecuteClusterQuery(GetQuery(cxt), "KUSTO_CLUSTER_NAME", "KUSTO_DB_NAME", null, "GetQuery"),
```
Comment (nmallick1) [inline data/templates/Detector_StaticSites.csx:23]: static apps is Antares, shouldn't they use ExecuteQuery syntax?

Comment (khaled-zayed) [reply data/templates/Detector_StaticSites.csx]: Good catch, updated 😊

**Review by nmallick1 — APPROVED** [review]

File: data/templates/Detector_StaticSites.csx (hunk 1 above)
Comment (ShekharGupta1988) [reply data/templates/Detector_StaticSites.csx]: But one thing to note here is - this is onboarded as an ARM Resource. So the Operation Context will not have the stamp in it. Hence the detector authors will have to figure out the stamp of the site (using observer) before invoking ExecuteQuery(..) method.

File: data/templates/Detector_StaticSites.csx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+
+[ArmResourceFilter(provider: "Microsoft.Web", resourceTypeName: "staticSites")]
+[Definition(Id = "YOUR_DETECTOR_ID", Name = "", Author = "YOUR_ALIAS", Description = "")]
+public async static Task<Response> Run(DataProviders dp, OperationContext<ArmResource> cxt, Response res)
+{
+    res.Dataset.Add(new DiagnosticData()
+    {
+        Table = await dp.Kusto.ExecuteQuery(GetQuery(cxt), cxt.Resource.Stamp.Name, null, "GetQuery"),
```
Comment (ShekharGupta1988) [inline data/templates/Detector_StaticSites.csx:23]: cxt.Resource will not have Stamp property here

Comment (ShekharGupta1988) [reply data/templates/Detector_StaticSites.csx:23]: I suggest in the template, we can add a line to fetch the stamp name via observer

Comment (hforeste) [reply data/templates/Detector_StaticSites.csx:23]: From what I learned, staticwebsites does not have a concept of stamp. Our infrastructure where we determine kusto cluster by stampname will not apply for staticwebsites.

Comment (khaled-zayed) [reply data/templates/Detector_StaticSites.csx:23]: Let's confirm with Mitch then first, thank you all.

**Pull request merged**